CRM/Utils/Hook/Drupal8.php

```diff
@@ -32,4 +32,13 @@
  */
 class CRM_Utils_Hook_Drupal8 extends CRM_Utils_Hook_DrupalBase {
 
+  /**
+   * {@inheritdoc}
+   */
+  protected function getDrupalModules() {
+    if (class_exists('\Drupal')) {
```

Comment:
I think this should be changed to:

```
if (class_exists('\Drupal') && \Drupal::hasContainer()) {
```

Otherwise, I'm seeing this line crash when running 'cv' on a Drupal 8 site because this gets called before 'cv' has a chance to bootstrap Drupal